projects/packages/codesniffer/check-docs.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# Ensure consistent sorting.
+export LC_ALL=C.UTF-8
+
+BASE=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+
+cd "$BASE"
+EXIT=0
+while IFS= read -r FILE; do
+	SNIFF=${FILE%Sniff.php}
+	SNIFF=${SNIFF/\/Sniffs\//\/}
+	SNIFF=${SNIFF//\//.}
+	if [[ ! -f "docs/$SNIFF.md" ]]; then
+		EXIT=1
+		echo "File $FILE presumably defines $SNIFF. Please create docs/$SNIFF.md to document it."
+	else
+		F="docs/$SNIFF.md"
```

Comment:
Could define this before the `if` if desired, since we're using it in both cases.

Reply:
Good call. f03aed7443